cpp/deplex/CMakeLists.txt

```diff
@@ -72,6 +72,10 @@ target_include_directories(
 target_link_libraries(${TARGET_NAME} PUBLIC Eigen3::Eigen)
 target_link_libraries(${TARGET_NAME} PRIVATE dsyev)
 target_link_libraries(${TARGET_NAME} PRIVATE rtl)
+
+set(PLATFORM_LINKFLAGS "-L${mimalloc_BINARY_DIR}/libmimalloc.dylib" "mimalloc")
```

Comment:
Why do we need this line? 
[mimalloc readme](https://github.com/microsoft/mimalloc#using-the-library) shows it is sufficient to use `target_link_libraries`